05-online-mentoring.md

```diff
@@ -0,0 +1,25 @@
+# Online Mentoring
+
+---
+
+## Tips
+* Prepare like you would for in-person mentoring. Make sure to go over the content
+the mentee is going to be working on.
+* Introduce yourself! Send a message on the slack channel or do a quick video
+meeting introducing yourself to your mentees. This is a great way to become
+familiar and comfortable around each other.
+* Send check in messages. It is difficult to see if someone is struggling
+with their work while online mentoring, so it's a good idea to check in and
+make sure everything is going smoothly.
+* Encourage screen share and video meetings. If a mentee needs help try to facilitate
+discussion through slack video meetings and screen share. Discourage the use of
+workspace sharing extensions such as CodeTogether and LiveShare, so the mentee can
+navigate the screen themselves.
+* Be accessible. Try to be available to respond quickly to any questions that arise.
+It is perfectly fine to step away from your computer and take a break,
+but make sure to inform your mentees if you are stepping away for a while. If you need
```

Comment:
Maybe add a suggestion here to utilize whatever online status indicators are available to them?